SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -12,11 +12,15 @@
 
 #include "AssetRegistryModule.h"
 #include "AssetDataTagMap.h"
+#include "Framework/Notifications/NotificationManager.h"
 #include "GeneralProjectSettings.h"
+#include "Internationalization/Regex.h"
+#include "Misc/FileHelper.h"
 #include "Misc/ScopedSlowTask.h"
 #include "SpatialGDKEditorSettings.h"
 #include "UObject/StrongObjectPtr.h"
 #include "Settings/ProjectPackagingSettings.h"
+#include "Widgets/Notifications/SNotificationList.h"
```

Comment:
Corresponds to: https://github.com/improbableio/UnrealEngine/blob/4.22-SpatialOSUnrealGDK-release/Engine/Source/Runtime/Slate/Public/Widgets/Notifications/SNotificationList.h

Had to check because it looks like a typo